api/api/apps.py

```diff
@@ -6,4 +6,4 @@ class ApiConfig(AppConfig):
     default_auto_field = "django.db.models.AutoField"
 
 
-# Comment to force CI
+# Comment to force CI. Don't forget to delete before merging.
```

Comment:
What does this mean? When should these comments (with the one in `ingestion_server/gunicorn.conf.py`) be removed?

Reply:
They can be removed now. Removing them will trigger the CI as well :slightly_smiling_face:

Reply:
@krysal, I removed the comments since we've checked that the CI passes.
Now this PR is ready for re-review.